nucypher/blockchain/eth/interfaces.py

```diff
@@ -285,8 +285,9 @@ def attach_middleware(self):
         # so we use external gas price oracles, instead (see #2139)
         if isinstance(self.client, InfuraClient):
             gas_strategy = datafeed_fallback_gas_price_strategy
+            self.gas_strategy = 'fast'  # FIXME
```

Comment:
Confused by this line...what's the reasoning here?

Reply:
The reasoning is that the `datafeed_fallback_gas_price_strategy` uses the datafeeds' defaults, which are all fast-like.

Reply:
I'm more curious about the reasoning for `FIXME`. Could you add some explanation?

Reply:
This is a monkeypatch to temporarily force `InfuraClients` to use the fast gas strategy until the datafeed strategies can be a bit more useful.